app/src/main/java/com/github/gouravkhunger/quotesapp/ui/fragments/QuoteFragment.kt

```diff
@@ -61,6 +65,18 @@ class QuoteFragment : Fragment(R.layout.fragment_quote) {
         // get the viewmodel and set observers on data
         viewModel = (activity as QuotesActivity).viewModel
 
+
+
+        shareQuote.setOnClickListener {
+            ShareUtil.share(
+                quoteCard,
+                requireContext(),
```

Comment:
Can you please hide the share button before sending the view to the share util class, this would make it better as the share button shouldn't be there in the formed image.